Bixal/move.mil
# Remove body field and its instances - **DO NOT MERGE/DEPLOY UNTIL TUESDAY 5/29**

Executes Pivotal ticket [157878318, removing the orphaned body field](https://www.pivotaltracker.com/story/show/157878318). 

**NOTE:** Feld has said he won't be able to move the content affected by this change until Monday, so he requested that we not merge it into staging until **Tuesday 5/29**.

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Contains config removing the body field from content types where it had existed (Basic Page and Article)
- Adds a css quick-fix fallback that keeps the body fields from displaying even if content still exists. This fallback will be removed as part of a future pull request, but it ensures we will be able to properly QA spacing and alignment of surrounding fields on staging.

## Testing

To verify the changes proposed in this pull request…

1. Pull code
1. Import config
1. Compile theme assets
1. Body fields, and the placeholders we had remaining on some pages (esp. in Tools & Resources section), should no longer appear either to the user or in the markup. 
1. Log in and edit the Basic Page content type. The body field should no longer exist in the list of fields.

## Screenshots

Content type admin (fixed on local):
![screen shot 2018-05-25 at 10 20 30 am](https://user-images.githubusercontent.com/29130580/40549380-4ae6b54c-6005-11e8-8de4-4920d7855fb5.png)
